recipes/wglext/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.20200813":
+    url: "https://raw.githubusercontent.com/KhronosGroup/OpenGL-Registry/main/api/GL/wglext.h"
```

Comment:
Perhaps `https://github.com/KhronosGroup/OpenGL-Registry/raw/43aad35cba1b76ea1f33f8d6234fa57ae8e2fbbe/api/GL/wglext.h` is better? Does not use that `githubusercontent.com` domain and is versioned.